packages/synthetix-main/contracts/modules/core/LiquidationModule.sol

```diff
@@ -140,24 +140,24 @@ contract LiquidationsModule is
 
         int rawVaultDebt = _vaultDebt(poolId, collateralType);
 
-        uint vaultDebt = rawVaultDebt < 0 ? 0 : uint(rawVaultDebt);
+        uint getVaultDebt = rawVaultDebt < 0 ? 0 : uint(rawVaultDebt);
```

Comment:
Isn't "get" weird in variable names?

To me it implies an action, so its good for function names.